src/sidebar/components/selection-tabs.js

```diff
@@ -1,49 +1,174 @@
-      return this.selectedTab === this.TAB_NOTES && this.totalNotes === 0;
-    };
-
-    this.showSidebarTutorial = function() {
-      return sessionUtil.shouldShowSidebarTutorial(session.state);
-    };
-  },
-  bindings: {
-    isLoading: '<',
-    isWaitingToAnchorAnnotations: '<',
-    selectedTab: '<',
-    totalAnnotations: '<',
-    totalNotes: '<',
-    totalOrphans: '<',
-  },
-  template: require('../templates/selection-tabs.html'),
+  const showNotesUnavailableMessage = function() {
+    return selectedTab === uiConstants.TAB_NOTES && totalNotes === 0;
+  };
+
+  const showSidebarTutorial = function() {
+    return sessionUtil.shouldShowSidebarTutorial(session.state);
+  };
+
+  return (
+    <div>
+      <div
+        className={classnames('selection-tabs', {
+          'selection-tabs--theme-clean': isThemeClean,
+        })}
+      >
+        <a
+          className={classnames('selection-tabs__type', {
+            'is-selected': selectedTab === uiConstants.TAB_ANNOTATIONS,
+          })}
+          href="#"
+          onClick={selectTab.bind(this, uiConstants.TAB_ANNOTATIONS)}
+          onTouchStart={selectTab.bind(this, uiConstants.TAB_ANNOTATIONS)}
+        >
+          Annotations
+          {totalAnnotations > 0 && !isWaitingToAnchorAnnotations && (
+            <span className="selection-tabs__count">{totalAnnotations}</span>
+          )}
+        </a>
+        <a
+          className={classnames('selection-tabs__type', {
+            'is-selected': selectedTab === uiConstants.TAB_NOTES,
+          })}
+          href="#"
+          onClick={selectTab.bind(this, uiConstants.TAB_NOTES)}
+          onTouchStart={selectTab.bind(this, uiConstants.TAB_NOTES)}
+        >
+          Page Notes
+          {totalNotes > 0 && !isWaitingToAnchorAnnotations && (
+            <span className="selection-tabs__count">{totalNotes}</span>
+          )}
+        </a>
+        <a
+          className={classnames(
+            'selection-tabs__type selection-tabs__type--orphan',
+            {
+              'is-selected': selectedTab === uiConstants.TAB_ORPHANS,
+            }
+          )}
+          href="#"
+          onClick={selectTab.bind(this, uiConstants.TAB_ORPHANS)}
+          onTouchStart={selectTab.bind(this, uiConstants.TAB_ORPHANS)}
+        >
+          Orphans
+          {totalOrphans > 0 && !isWaitingToAnchorAnnotations && (
+            <span className="selection-tabs__count">{totalOrphans}</span>
+          )}
+        </a>
+      </div>
+      {/*
+        uiConstants.TAB_NOTES
+        {settings.enableExperimentalNewNoteButton && selectedTab === uiConstants.TAB_NOTES &&
+          <new-note-btn></new-note-btn>
+        }
+        */}
+      {!isLoading() && (
+        <div className="selection-tabs__empty-message">
+          {showNotesUnavailableMessage() && (
+            <div className="annotation-unavailable-message">
+              <p className="annotation-unavailable-message__label">
```

Comment:
Is this `p` element truly necessary? The answer may be yes. Some of the markup here feels fiddly.